src/T2DMap.cpp

```diff
@@ -583,25 +584,22 @@ void T2DMap::initiateSpeedWalk(const int speedWalkStartRoomId, const int speedWa
     }
 }
 
-// This has been refactored to a separate function out of the paintEven() code
-// because we need to use it in two places - one for every room that is not the
-// player's room and then, AFTER all those have been drawn, once for the
-// player's room if it is visible. This is so it is drawn LAST (and any effects,
-// or extra markings for it do not get overwritten by the drawing of the other
-// rooms)...
```

Comment:
@vadi2 You think this is unhelpful then?

Reply:
Yeah, while the function's history can be interesting, it might not always be necessary for using it.